burn-book/src/import/pytorch-model.md

```diff
@@ -230,7 +230,8 @@ Which produces the following weights structure (viewed in
 
 You can use the `PyTorchFileRecorder` to change the attribute names and the order of the attributes
 by specifying a regular expression (See
-[regex::Regex::replace](https://docs.rs/regex/latest/regex/struct.Regex.html#method.replace)) to
+[regex::Regex::replace](https://docs.rs/regex/latest/regex/struct.Regex.html#method.replace) and
+[try it online](https://rregex.dev/?version=1.10&method=replace)) to
```

Comment:
Oh very cool I didn't know there was an online sandbox to mess around with regex. I always used regex101 but it's not a 1-to-1 due to string formatting.